airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/DatabaseConfigPersistenceLoadDataTest.java

```diff
@@ -100,23 +101,23 @@ public void testUpdateConfigsInNonEmptyDatabase() throws Exception {
   @DisplayName("When a connector is in use, its definition should not be updated")
   public void testNoUpdateForUsedConnector() throws Exception {
     // the seed has a newer version of s3 destination and github source
-    StandardDestinationDefinition destinationS3V2 = YamlSeedConfigPersistence.get()
+    final StandardDestinationDefinition destinationS3V2 = YamlSeedConfigPersistence.get()
```

Comment:
This test shows a good example why the static class variable should not be modified. It is very confusing that we need to call `initialize` in the `setup` method first before calling the `get` here. No one will remember to do this until they get the exception message.